app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ] Please create a method for the like model which updates the comments counter for a post,as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md#:~:text=A%20method%20that%20updates%20the%20likes%20counter%20for%20a%20post.) requirement.